app/src/main/index.js

```diff
@@ -266,36 +262,6 @@ async function initBaseserver (chainId, home, node) {
   await expectCleanExit(child, 'gaia init exited unplanned')
 }
 
-async function backupData (root) {
```

Comment:
why don't we need to backup anymore?

Reply:
We were previously backing up data when we switched networks, since we had to re-init and we didn't want to just delete the old data. Now we use separate folders for different networks, so we shouldn't ever have to do this.

Reply:
makes sense!